musthe/musthe.py

```diff
@@ -379,6 +394,30 @@ def __eq__(self, other):
         else:
             return all(self.notes[i] == other.notes[i]
                        for i in range(len(self.notes)))
+    
+    def lilypond_notation(self, duration: Union[str,int] = ""):
+        """Returns the chord as string that can be used in a LilyPond
+        ``\chordmode`` block.
```

Comment:
can you document what the "duration" parameter means?

Reply:
This should be fixed in the updated pr.